includes/CLI/Plugin_Check_Command.php

```diff
@@ -77,6 +78,7 @@ public function __construct( Plugin_Context $plugin_context ) {
 	 *   - table
 	 *   - csv
 	 *   - json
+	 *   - wporg
```

Comment:
Add new option in docblock.

`Format to display the results. Options are table, csv, json, and wporg. The default will be a table.`